### ISSUE TITLE
Crash when leaving server

### ISSUE THREAD

**Issue by Wash113** [FrozenBlock/FrozenLib#47]
Getting a crash when exiting a server, only lead is a reference to the frozenblock lib in the stacktrace. 

**Steps To Reproduce**
1. Click disconnect on server
2. 
3. 
4. 

**Expected Behavior**
Leaves server and returns to main menu

**Screenshots**

**Logs**
https://mclo.gs/8dyydGx

**Additional context**
Ive tried this on both the latest versions of Quilt and Fabric. Issue persists across multiple computers

**Comment by TheDrawingCoder-Gamer** [comment]
yep, can confirm. after figuring out my install traced it back here

**Comment by Treetrain1** [comment]
Please provide the full log file

**Comment by TheDrawingCoder-Gamer** [comment]
https://mclo.gs/Lix2Z7s
I could not for the life of me find a minimum set that causes a crash on the unknown registry screen. The intersection of my and OPs mods doesn't crash

**Comment by Treetrain1** [comment]
> https://mclo.gs/Lix2Z7s I could not for the life of me find a minimum set that causes a crash on the unknown registry screen. The intersection of my and OPs mods doesn't crash

Can you provide the full log as the crash report doesn't give any useful details

**Comment by TheDrawingCoder-Gamer** [comment]
https://mclo.gs/VmNUyO9

**Comment by TheDrawingCoder-Gamer** [comment]
this only happens if the server is modded? it's possible this is an issue because of the server as well. I can dump logs of the server too

**Comment by TheDrawingCoder-Gamer** [comment]
[latest.log](https://github.com/user-attachments/files/16431994/latest.log)

**Comment by AViewFromTheTop** [comment]
Does this still happen?

**Comment by AViewFromTheTop** [comment]
...
